docs/requirements.txt

```diff
@@ -1,2 +1,4 @@
 sphinx>=5.0.0
 breathe>=4.30.0
+sphinx-tabs>=3.4.1
+z3-solver>=4.12.1
```

Comment:
I believe we don't require such a high version of Z3 :thinking: I could be wrong, but `>=4.8` should work fine.